content/doc/book/pipeline/pipeline-best-practices.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: section
+title: Pipeline Best Practices
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author: Alex Taylor
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+:toc:
+endif::[]
+
+= Applying best practices to Pipelines
+
+== General
+
+=== Using Groovy code in Pipelines as glue
```

Comment:
@alexanderrtaylor @HCloward 
If this is a "how to" guide then `Using...` works here.  If this is set of guidelines, I'd rather we use imperative `Use Groovy code in Pipelines as glue`.   But it depends on what we're shooting for.